### ISSUE TITLE
addLine doesn't work with ModelCoefficients

### ISSUE THREAD

**Issue by VictorLamoine** [PointCloudLibrary/pcl#1029]
[addLine](http://docs.pointclouds.org/trunk/classpcl_1_1visualization_1_1_p_c_l_visualizer.html#a5fd2b99b85da61df8b58980bdb6b0f41) doesn't work when used with ModelCoefficients
## Test code

``` cpp
#include <pcl/common/common.h>
#include <pcl/visualization/pcl_visualizer.h>

int
main (int argc,
      char** argv)
{
  pcl::visualization::PCLVisualizer viewer ("Line");
  pcl::ModelCoefficients coeff;
  coeff.values.push_back (0);
  coeff.values.push_back (0);
  coeff.values.push_back (0);
  coeff.values.push_back (1);
  coeff.values.push_back (0);
  coeff.values.push_back (0);

  viewer.addLine (coeff, "line");
  while (!viewer.wasStopped ())
  {
    viewer.spinOnce ();
  }

  return 0;
}
```
## gcc error

``` sh
In file included from /usr/local/include/pcl-1.8/pcl/visualization/pcl_visualizer.h:2200:0,
                 from /home/dell/Copy/test_pcl/src/test_pcl.cpp:2:
/usr/local/include/pcl-1.8/pcl/visualization/impl/pcl_visualizer.hpp: In instantiation of ‘bool pcl::visualization::PCLVisualizer::addLine(const P1&, const P2&, double, double, double, const string&, int) [with P1 = pcl::ModelCoefficients; P2 = char [5]; std::string = std::basic_string<char>]’:
/usr/local/include/pcl-1.8/pcl/visualization/impl/pcl_visualizer.hpp:578:11:   required from ‘bool pcl::visualization::PCLVisualizer::addLine(const P1&, const P2&, const string&, int) [with P1 = pcl::ModelCoefficients; P2 = char [5]; std::string = std::basic_string<char>]’
make[2]: quittant le répertoire « /home/dell/Copy/test_pcl/build »
/home/dell/Copy/test_pcl/src/test_pcl.cpp:17:32:   required from here
/usr/local/include/pcl-1.8/pcl/visualization/impl/pcl_visualizer.hpp:464:94: error: ‘const struct pcl::ModelCoefficients’ has no member named ‘getVector4fMap’
make[1]: quittant le répertoire « /home/dell/Copy/test_pcl/build »
   vtkSmartPointer<vtkDataSet> data = createLine (pt1.getVector4fMap (), pt2.getVector4fMap ());
                                                                                              ^
/usr/local/include/pcl-1.8/pcl/visualization/impl/pcl_visualizer.hpp:464:94: error: request for member ‘getVector4fMap’ in ‘pt2’, which is of non-class type ‘const char [5]’
make[2]: *** [CMakeFiles/test_pcl.dir/test_pcl.cpp.o] Erreur 1
make[1]: *** [CMakeFiles/test_pcl.dir/all] Erreur 2
make: *** [all] Erreur 2
```

Using `viewer.addLine (coeff);` solves the problem.

[Reported by pawkro](http://www.pcl-users.org/NN-to-a-circle-drawing-circle-in-PCL-tp4036336p4036412.html)

**Comment by jpmerc** [comment]
I had the same problem and found this issue. I have solved the problem by doing this : 
```c
viewer.addLine (coeff, "line"); // does not work 
viewer.addLine (coeff, std::string("line")); // works
```

**Comment by seanngpack** [comment]
is the overloaded addLine() method that takes in pcl::ModelCoefficients deprecated in pcl 1.10? I can only see the signatures that take in pt1 and pt2

**Comment by kunaltyagi** [comment]
We're having trouble with VTK and OpenGL on MacOS, because of which homebrew hasn't shipped 1.10. Any help on these front will allow PCL 1.11 to be released on MacOS too. Meanwhile, if you don't need surface module, feel free to switch off surface (and visualization) and compile from source
